### ISSUE TITLE
Play button out-of-screen on Nexus 4

### ISSUE THREAD

**Issue by leoarnold** [gpodder/gpodder-android#9]
With the setup
- Device: Google Nexus 4
- OS: Android 5.1.1
- App: gPodder 4.6.0
- Podcast: any episode with chapters (e.g. [FS166](http://freakshow.fm/fs166-schleifenlaengenvergleich))
- View: "Now Playing" in portrait

the "Chapters" button will be visible, pushing the "Play / Pause" button beyond the left screen edge into invisibility.

One can still access all buttons, when switching to landscape view.

**Comment by thp** [comment]
Indeed, this needs some tweaking to the QML to make it work for more screen resolutions/ratios.
